client/src/app/shared/components/head-bar/head-bar.component.ts

```diff
@@ -71,6 +71,12 @@ export class HeadBarComponent {
     @Input()
     public editMode = false;
 
+    /**
+     * The save button can manually be hidden.
+     */
+    @Input()
+    public displaySaveButton = true;
```

Comment:
Hmm, hidden and disabled i something different. Please be clear with the name of this input.